src/wp-includes/wp-html.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
IMO this file is not needed and should be removed.

Instead, move each `require_once` to `wp-settings.php`. Why? It's consistent with how Core loads its APIs including REST, Sitemaps, etc.

Reply:
Commit https://github.com/WordPress/wordpress-develop/pull/3920/commits/b708c6b618770c7e80981c93c0bd2ab9d73a7819 

* Loads the APIs files directly from `wp-settings.php` and uses `require` instead of `require_once` > for consistency with how Core loads its files
* Updates test classes.

A follow-up commit coming to remove loading the files in the test classes.

Reply:
https://github.com/WordPress/wordpress-develop/pull/3920/commits/521a50016287a5eeced40a85684a479d23690e19 removes loading the API files in the test classes. Why? The files are already loaded into memory during test suite bootstrap.